File: tests/benchmarks/find_primes/find_primes.js

const n = 10000;

function is_prime(n) {
    if(n == 0 || n == 1) {
        return false;
    }
    for(let i = 2; i < n; ++i) {
        if(n % i == 0) {
            return false;
        }
    }
    return true;
}

for(let i = 0; i < n; ++i) {
    if(is_prime(i)) {
        // console.log(i);
    }
}

File: tests/benchmarks/find-primes/find-primes.lua

local n = 10000

local function is_prime(n)
    if n == 0 or n == 1 then
        return false
    end
    for i = 2, n - 1 do
        if n % i == 0 then
            return false
        end
    end
    return true
end

for i = 0, n - 1 do
    if is_prime(i) then
        -- print(i)
    end
end
